### ISSUE TITLE
TFLite-Micro Operation Support for "Dropout"

### ISSUE THREAD

**Issue by galactica1983** [tensorflow/tensorflow#39300]
@tensorflow/micro

**System information**
- Host OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
N/A
- TensorFlow installed from (source or binary):
N/A
- Tensorflow version (commit SHA if source):
N/A
- Target platform (e.g. Arm Mbed OS, Arduino Nano 33 etc.):
N/A

**Describe the problem**
Hi, may I know the plan to support "Dropout" operator in TFLite-Micro?
Thanks for your help!

**Please provide the exact sequence of commands/steps when you ran into the problem**

**Comment by tilakrayal** [comment]
@galactica1983,
Could you please let us know in what cases you are using dropout and provide the Usecase. Please try the workaround as mentioned below.

1. Try to set the training flag in **Dropout** layer when building keras model (this enables Dropout all the time)

`x = Dropout(0.1)(x, training=True)`

2. Set TFlite converter Flags as mentioned
```
converter = tf.lite.TFLiteConverter.from_keras_model(actor.model)
converter.target_spec.supported_ops = [
                tf.lite.OpsSet.TFLITE_BUILTINS,  # enable TensorFlow Lite ops.
                tf.lite.OpsSet.SELECT_TF_OPS  # enable TensorFlow ops.
            ]
tflite_model = converter.convert()

open("converted_model.tflite", "wb").write(tflite_model) 
```
 When we are using this, the model load time will be increasing. Also please take a look at this [issue](https://github.com/tensorflow/tensorflow/issues/44232) for reference. Thank you!
